templates/web/pages/bmviii_apply.tmpl

```diff
@@ -0,0 +1,431 @@
+                                    {{ end }}
+                                </label>
+                            </div>
+                        </div>
+					</div>
+
+					<div class="field">
+                        <div class="field-label is-normal">
+                            <label for="is-first-hackathon" class="label">
+                                <p class="bmviii-app-font" style="margin-bottom: 0px">
+                                    Have you attended a hackathon before?
+                                </p>
+                            </label>
+                            <div class="field-body">
+                                <div class="field">
+                                    <div class="control">
+                                        <div class="select is-fullwidth">
+                                            <select name="is-first-hackathon" style="background-color: #F0F0F0" required>
+                                                {{ if not .FormRefill.IsFirstHackathon }}
+                                                <option value="off" selected="true">No</option>
+                                                <option value="on">Yes</option>
+                                                {{ else }}
+                                                <option value="off">No</option>
+                                                <option value="on" selected="true">Yes</option>
+                                                {{ end }}
+                                            </select>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+					</div>
+
+					<h3 style="font-family: Cabin">4. Short Answer Questions</h3>
+
+					<div class="field">
+						<label for="why-bm" class="label">
+						    <p class="bmviii-app-font" style="margin-bottom: 0px">
+						        Why do you want to come to BoilerMake? (100 word limit)
+                            </p>
+                        </label>
+						<div class="control">
+							<textarea class="textarea" style="background-color: #F0F0F0" name="why-bm" required>{{ .FormRefill.WhyBM }}</textarea>
+						</div>
+					</div>
+
+                    <div class="field">
+                        <label for="proj-idea" class="label">
+                            <p class="bmviii-app-font" style="margin-bottom: 0px">
+                                Do you already have a project idea or technology that you're interested in learning? (It's okay if you don't) (100 word limit)
+                            </p>
+                        </label>
+                        <div class="control">
+                            <textarea class="textarea" style="background-color: #F0F0F0" name="proj-idea" required>{{ .FormRefill.ProjIdea }}</textarea>
```

Comment:
Should we not make this `required` so people who don't have a project idea can just leave it blank?

Reply:
Currently in discussion

Reply:
Gonna be optional